src/index.html

```diff
@@ -84,7 +84,7 @@
                  data-chart-title="Number of loans originated"
                  data-chart-color="blue"
                  data-chart-description="This is the chart description."
-                 data-chart-source="consumer-credit-trends/num_data_AUT.csv">
+                 data-chart-source="consumer-credit-trends/auto-loan/num_data_AUT.csv">
```

Comment:
Looks like this would mean a change to the script. Pinging @hillaryj for awareness, so she can work with you.